go.mod

```diff
@@ -1,3 +0,0 @@
-module github.com/datumbrain/jcsv
-
-go 1.16
```

Comment:
Why is this file deleted?

Reply:
I had to create my own mod file, I could not execute my main function without doing so.